src/v2/BaseLoginRouter.js

```diff
@@ -86,7 +86,7 @@ export default Router.extend({
     // TODO: OKTA-244631 How to suface up the CORS error in IDX?
     // Since in new pipeline, it invokes introspect API first
     // hence no way to call GlobalError when CORS failure.
-    this.listenTo(this.appState, 'change:remediationFailure', function (appState, err) {
+    this.listenTo(this.appState, 'change:introspectError', function (appState, err) {
```

Comment:
Is this because we are handling remediation failures in the form controller via `updateAppStateWithNewIdx ` ?

Reply:
Not really. This probably never worked 
- I didn't find where fire `remediationFailure` (even at oie initial commit) 🤦 
- as v2 router didn't take any `introspectError` b/c when `introspectError`, we fall back to v1. That's why `TODO OKTA-244631`.